torch/csrc/jit/passes/onnx/constant_fold.cpp

```diff
@@ -295,7 +299,7 @@ c10::optional<at::Tensor> runTorchBackendForOnnx(
         inputTensorValues[0], p, node->is(attr::axes), node->i(attr::keepdims));
     return c10::optional<at::Tensor>(updated_val);
   } else if (node->kind() == onnx::Gather) {
-    assert(inputTensorValues.size() == 1);
+    assert(inputTensorValues.size() == 2);
```

Comment:
why change this?

Reply:
gather has 2 inputs, that's why my tests were failing

Reply:
This was a bug in gather implementation for constant folding. This node is required to have 2 inputs always (Note that this assertion as not blocking tests on CI, and was caught locally).

Reply:
@houseroad if you have no further comments, can this PR be merged?